rune/libenclave/internal/runtime/pal/skeleton/sgxsign.c

```diff
@@ -295,13 +302,18 @@ static bool measure_encl(const char *path, uint8_t *mrenclave,
 	}
 
 	struct sgx_tcs *tcs = bin;
-	tcs->ssa_offset = sb.st_size;
-
+	/* SSA frame is located behind encl.bin */
+	tcs->ssa_offset = encl_offset + sb.st_size;
```

Comment:
Offset of the base of the State Save Area stack, relative to the enclave base.Must be page aligned.